src/sentry/integrations/github_enterprise/webhook.py

```diff
@@ -36,6 +36,7 @@ def get_installation_metadata(event, host):
             provider="github_enterprise",
         )
     except Integration.DoesNotExist:
+        logger.exception()  # This creates an error in Sentry
```

Comment:
I don't think the comment is necessary, this is standard usage. I'd usually recommend you include a message to give a little context, but it's not required.